docs/user-docs/latest/plugins/installing-plugins-via-marketplace.md

```diff
@@ -4,37 +4,41 @@ description: Instructions to install new plugins in a Strapi application via the
 canonicalUrl: https://docs.strapi.io/user-docs/latest/plugins/installing-plugins-via-marketplace.html
 ---
 
-# Installing plugins via the Marketplace
+# Marketplace plugins
 
-::: callout 🚧 The Marketplace is currently not available for v4. It will be back soon, stay tuned!
-In the meantime, plugins can be installed via the Command Line Interface (see [Developer Documentation](/developer-docs/latest/developer-resources/cli/CLI.md#strapi-install)).
-:::
-
-The Marketplace is a section of the admin panel that lists all plugins that can be installed in a Strapi application. The Marketplace is accessible from ![Marketplace icon](../assets/icons/marketplace.svg) _Marketplace_ in the main navigation of the admin panel.
+The Marketplace contains optional plugins built to customize the core Strapi installation. The Marketplace is located in the admin panel, indicated by the ![Marketplace icon](../assets/icons/marketplace.svg) _Marketplace_. Marketplace plugins are installed using the CLI (see [Developer Documentation](/developer-docs/latest/developer-resources/cli/CLI.md#strapi-install)).
```

Comment:
I think it would be nice to add a bit more information here, such as:
- what actions are available from that interface,
- what the reader is going to learn by reading this documentation section, etc.

Also, the link to the dev docs for more information, I think I would move it to the "Install plugin" section, seems more useful there in my opinion. We can rework this part together if you want 😄

Reply:
I made some revisions based on your suggestions. Though I am not quite sure it is what you meant.